DEVELOPMENT.md

```diff
@@ -0,0 +1,496 @@
+# KCP-Edge Contributor Development Environment
+
+## Required Packages:
+   - ko: https://ko.build/install/ 
+   - gcc
+   - docker 
+   - jq
+   - make
+   - go (version expected 1.19)
+   - kind
+   - kubectl (version range expected: 1.23-1.25)
+
+
+## Supported OS Platforms 
+  - Linux
+  - MacOS
+  - Windows WSL/Ubuntu
+
+
+After cloning this repo (see quickstart), run the following script to install the required package (Linux or MacOS):
+
+```bash
+hack/install_req.sh
+```
+
+On MacOS you could use the following command as an alternative to the above:
+
+```bash
+brew install ko gcc jq make go@1.19 kind kubectl
+```
+
+For Windows WSL/Ubuntu platform, follow the instructions [here](docs/wls_ubuntu_platform.md)
+
+
+## Setup Development Environment
+
+### 1. Clone the kcp-edge repo:
+
+```shell
+git clone https://github.com/kcp-dev/edge-mc.git
+```
+
+
+### 2. Build the kcp-edge binaries:
+
+```shell
+cd edge-mc
+make build
+```
+
+The kcp-edge binaries is created inside the `./bin` directory. Add this directory to your $PATH:
+
+```shell
+export PATH="$PATH:$(pwd)/bin"
+```
+
+
+### 3. Create your own edge infrastructure (edge clusters) - Bring Your Own Clusters (BYOC)
+
+Create your edge clusters or bring your own k8s edge clusters. In this example, we deploy only a single cluster using [kind](https://kind.sigs.k8s.io/). Our edge cluster is named “florin”:
+
+```shell
+kind create cluster --name florin
+```  
+
+### 4. Install KCP
```

Comment:
I think that at this point the reader should be referred to the user documentation.
Attempting to maintain a copy here adds work for us and opportunities to make mistakes.
Having a copy of a large volume of material here also confuses the developer: she wonders why she is reading something different from users --- is there a reason?  Is there an important difference that she is overlooking?